README.md

```diff
@@ -2,15 +2,15 @@
 
 # Portfolio setup and mobile version skeleton
 
-> In this project I built the various sections of my desktop version portfolio.
+> In this project I built the various sections of my portfolio.
 
 ## Built With
 
 - Major languages: HTML and CSS
 
-## Live Demo (if available)
+## Live Demo
 
-![Image](absolute link)
+![Image](https://krisbobo.github.io/Portfolio-mobileVersionSkeleton/)
```

Comment:
You should change the `Image` text to something like `Live Demo` because this is where I am currently being redirected to when I click on the README link. 

<img width="297" alt="Screen Shot 2022-06-06 at 10 22 20" src="https://user-images.githubusercontent.com/82426682/172191781-41b6c398-f0d1-4a81-84a7-c38ab1af33f3.png">

I did check the actual working link and everything looks perfect, so your project should be ready for approval after you fix your README link. Please do so.